File: Core Graphics/5-Dynamic-Bezier-Shapes/DynamicShapesBasics_ObjectiveC/DynamicShapesBasics/ButtonView.m
//
//  ButtonView.m
//  DynamicShapesBasics
//
//  Created by Peter Krajčík on 4/28/14.
//  Copyright (c) 2014 PixelCut. All rights reserved.
//

#import "ButtonView.h"

#import "MyStyleKit.h"

@implementation ButtonView

- (id)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        // Initialization code
    }
    return self;
}

- (void)drawRect: (CGRect)rect
{
	[MyStyleKit drawBubbleButtonWithFrame: self.bounds];
}

@end

File: Core Graphics/5-Dynamic-Bezier-Shapes/DynamicShapesBasics_Swift/DynamicShapesBasics/ButtonView.swift
//
//  ButtonView.swift
//  DynamicShapesBasics
//
//  Created by Matej Duník on 10/10/2018.
//  Copyright © 2018 Matej Duník. All rights reserved.
//

import UIKit

class ButtonView: UIView {

    override func draw(_ rect: CGRect) {
        // Drawing code
        MyStyleKit.drawBubbleButton(frame: self.bounds)
    }

}
